content/en/real_user_monitoring/browser/_index.md

```diff
@@ -171,25 +275,26 @@ Add the generated code snippet to the head tag of every HTML page you want to mo
   })
 </script>
 ```
+{{</ site-region>}}
 
 </details>
 
 <details>
   <summary>before<code>v4.30.0</code></summary>
 
-<!-- prettier-ignore -->
+{{< site-region region="us" >}}
 ```html
 <script>
   (function(h,o,u,n,d) {
      h=h[d]=h[d]||{q:[],onReady:function(c){h.q.push(c)}}
      d=o.createElement(u);d.async=1;d.src=n
      n=o.getElementsByTagName(u)[0];n.parentNode.insertBefore(d,n)
-  })(window,document,'script','https://www.datadoghq-browser-agent.com/datadog-rum-v4.js','DD_RUM')
+  })(window,document,'script','https://www.datadoghq-browser-agent.com/us1/v4/datadog-rum.js','DD_RUM')
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏ Those new URLs have been released in v4.35.0. At the v4.30.0 time, only the suffixed URL was available. Should we change those URLs in anterior code snippets?

Reply:
Indeed 🤔
However, those snippets would work as expected even with the new URLs.
I'd be in favour of promoting the new URLs everywhere to avoid confusion with different setups documented with or without site instead of adding a new `before 4.35.0` section to reflect that the new URLs has been introduced from there.
wdyt?

Reply:
Discuss IRL, let's use the new URLs everywhere to let customers focusing on parameters change between versions and not adding noise with CDN url changes